index.js

```diff
@@ -252,6 +259,32 @@ function setCookie (res, name, val, options) {
   res.setHeader('set-cookie', header)
 }
 
+/**
+ * Get secrect from set-cookie header in HTTP response.
+ *
+ * @param {OutgoingMessage} res
+ * @param {Object} [cookie]
+ * @api private
+ */
+
+function getSecertFromSetCookieHeader (res, cookie) {
+  let setCookieHeaders = res.getHeader("set-cookie")
+  if (setCookieHeaders) {
+    for (let setCookieHeader of setCookieHeaders) {
+      setCookieHeader = Cookie.parse(setCookieHeader)
```

Comment:
We should not be using `cookie.parse` to parse the `set-cookie` header, as it is only designed to parse the `cookie` header. I believe in your comments you pointed out the `secure` attribute issue, which is a great point. I believe you cannot access this attribute due to the usage of the wrong parsing API.